css/frontPage.css

```diff
@@ -143,3 +143,24 @@ footer {
     clear:             right;
     box-shadow:        4px 4px 2px rgba(0, 0, 0, 0.5);
 }
+
+
+
+.dropdown {
+    position: relative;
+    display: inline-block;
+}
+
+.dropdown-content {
+    display: none;
+    position: absolute;
+    left: 100%;
+    top: 10%;
+    background-color: #f9f9f9;
+    min-width: 160px;
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
+}
+
+.dropdown:hover .dropdown-content {
+    display: block;
```

Comment:
The hover elements seem to appear under the original list.  Maybe try using the z-index property.